packages/ember-runtime/tests/mixins/array_test.js

```diff
@@ -60,7 +60,7 @@ ArrayTests.extend({
 
   newObject(ary) {
     ary = ary ? ary.slice() : this.newFixture(3);
-    return new TestArray(ary);
+    return TestArray.create({ _content: ary });
```

Comment:
This is testing low level building blocks, this stuff isn't documented, I don't even remember which thing needs this set up this way.